modules/rpc/handler.go

```diff
@@ -28,56 +28,55 @@ import (
 	"go.uber.org/fx/internal/fxcontext"
 	"go.uber.org/fx/service"
 	"go.uber.org/thriftrw/wire"
-	"go.uber.org/yarpc"
+	"go.uber.org/yarpc/api/transport"
 	"go.uber.org/yarpc/encoding/thrift"
-	"go.uber.org/yarpc/transport"
 )
 
 // UnaryHandler is a wrapper for YARPC thrift.UnaryHandler
 type UnaryHandler interface {
-	Handle(ctx fx.Context, reqMeta yarpc.ReqMeta, body wire.Value) (thrift.Response, error)
+	Handle(ctx fx.Context, body wire.Value) (thrift.Response, error)
 }
 
 // UnaryHandlerFunc calls the YARPC Handle with fx.Context
-type UnaryHandlerFunc func(fx.Context, yarpc.ReqMeta, wire.Value) (thrift.Response, error)
+type UnaryHandlerFunc func(fx.Context, wire.Value) (thrift.Response, error)
 
 // Handle calls the caller HandlerFunc.
-func (f UnaryHandlerFunc) Handle(ctx fx.Context, reqMeta yarpc.ReqMeta, body wire.Value) (thrift.Response, error) {
-	return f(ctx, reqMeta, body)
+func (f UnaryHandlerFunc) Handle(ctx fx.Context, body wire.Value) (thrift.Response, error) {
+	return f(ctx, body)
 }
 
 // WrapUnary wraps the unary handler and returns a thrift.UnaryHandlerFunc for yarpc calls
 // TODO(anup): GFM-255 Remove host parameter once updated yarpc plugin is imported in the repo
```

Comment:
I'd like to leave that for @anuptalwalkar in another PR. Don't want to conflate many things together, there is some work needed to remove all those comments